code/game/atoms.dm

```diff
@@ -223,7 +222,7 @@
 
 /atom/proc/emp_act(severity)
 	SendSignal(COMSIG_ATOM_EMP_ACT, severity)
-	if(istype(wires) && !(flags_2 & NO_EMP_WIRES_2))
+	if(istype(wires) && !(flags_1 & NO_EMP_WIRES_1))

```

Comment:
I thought you were removing this

Reply:
I am, just in another PR, this one already has more than enough stuff